apps/sensenet/src/context/ThemeProvider.tsx

```diff
@@ -14,11 +16,19 @@ const mergeThemes = (options: ThemeOptions, type: 'light' | 'dark') =>
   })
 
 export const ThemeProvider: React.FunctionComponent<{ theme: ThemeOptions }> = props => {
-  const ps = useContext(ResponsivePersonalSetttings)
-  const theme = mergeThemes(props.theme, ps.theme)
+  const injector = useInjector()
+  const service = injector.getInstance(PersonalSettings)
+  const settings = service.userValue.getValue()
+  const [pageTheme, setPageTheme] = useState<Theme>(mergeThemes(props.theme, settings.theme))
```

Comment:
You should use personalSettings here instead of the service.userValue so we can avoid the useInjector, injector.getInstance... etc calls.